api/src/org/labkey/api/util/CPUTimer.java

```diff
@@ -129,6 +129,16 @@ public String getDuration()
     }
     
 
+    public float getAverage()
+    {
+        return (_calls == 0 ? 0 : (float)getTotalMilliseconds() / _calls);
```

Comment:
```suggestion
        return _calls == 0 ? 0 : (float)getTotalMilliseconds() / _calls;
```
Not necessary, of course.